docs/docs/cmd/entra/pim/pim-role-assignment-add.mdx

```diff
@@ -0,0 +1,221 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# entra pim role assignment add
+
+Request activation of an Entra ID role assignment for a user or group.
+
+## Usage
+
+```sh
+m365 entra pim role assignment add [options]
+```
+
+## Options
+
+```md definition-list
+`-n, --roleDefinitionName [roleDefinitionName]`	
+: Name of the role definition that should be assigned. Specify either `roleDefinitionName` or `roleDefinitionId` but not both.
+
+`-i, --roleDefinitionId [roleDefinitionId]`	
+: Id of the role definition that is being assigned. Specify either `roleDefinitionName` or `roleDefinitionId` but not both.
+
+`--userId [userId]`	
+: Id of the user that will be granted the assignment. Specify either `userId`, `userName`, `groupId` or `groupName`. If not specified, the current user will be used.
+
+`--userName [userName]`	
+: UPN of the user that will be granted the assignment. Specify either `userId`, `userName`, `groupId` or `groupName`. If not specified, the current user will be used.
+
+`--groupId [groupId]`
+: Id of the group that will be granted the assignment. Specify either `userId`, `userName`, `groupId` or `groupName`. If not specified, the current user will be used.
+
+`--groupName [groupName]`
+: Display name of the group that will be granted the assignment. Specify either `userId`, `userName`, `groupId` or `groupName`. If not specified, the current user will be used.
+
+`--administrativeUnitId [administrativeUnitId]`
+: Id of the administrative unit representing the scope of the assignment. Specify either `administrativeUnitId` or `applicationId`. If not specified, default scope will be tenant-wide.
+
+`--applicationId [applicationId]`
+: Object id of the application representing the scope of the assignment. Specify either `administrativeUnitId` or `applicationId`. If not specified, default scope will be tenant-wide.
+
+`-j, --justification [justification]`
+: An optional justification message.
+
+`-s, --startDateTime [startDateTime]`
+: When the assignment should start. If left out, the assignment will start from the current time.
+
+`-e, --endDateTime [endDateTime]`
+: When the assignment should end. Specify either `duration` or `endDateTime`.
+
+`-d, --duration [duration]`
+: How long the assignment should last. Write in ISO 8601 format for durations: PT3H for 3 hours.
+
+`--ticketNumber [ticketNumber]`
+: Optional ticket number value to communicate with the request.
+
+`--ticketSystem [ticketSystem]`
+: Optional ticket system to communicate with the request.
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To use this command you must be **Privileged Role Administrator**.
```

Comment:
Shouldn't this read that you need this role when activating for other users? I would find it strange if I needed this role if all I wanted was to request activation of my own role.